static/install

```diff
@@ -70,7 +70,7 @@ esac
 today=$(date +"%Y-%m-%d")
 tar_target_name="dune-$today-$target"
 tar_target="$tar_target_name.tar.gz"
-dune_tar_uri="https://get.dune.build/$today/$target/$tar_target"
+dune_tar_uri="https://preview.dune.build/latest/$target"
```

Comment:
Does this mismatch with `$today` when it is in a different timezone ?

Reply:
I thinking about downloading latest first and get `$today` information from the name of the file.

Reply:
We don't really care about `$today`, it's just some local file name that it gets saved into. It could also be fixed to be `tar_target=latest.tar.gz` and avoid `$today` completely.